### PR TITLE
Followup to 6896, add explanation of workaround.

### PR DESCRIPTION
**Pull request by jimfb**
Followup to #6896, add explanation of workaround.

### CODE REVIEW

**Comment by jimfb** [comment]
@gaearon

File: src/renderers/dom/shared/ReactDOMComponent.js [hunk 1]
```diff
@@ -624,6 +624,8 @@ ReactDOMComponent.Mixin = {
         } else if (props.is) {
           el = ownerDocument.createElement(this._currentElement.type, props.is);
         } else {
+          // Separate else branch instead of using `props.is || undefined` above becuase of a firefox bug.
```
Comment (gaearon) [inline src/renderers/dom/shared/ReactDOMComponent.js]: Typo: becuase
Nit: let’s capitalize Firefox :-)

**Comment by gaearon** [comment]
Let’s also add a link to https://bugzilla.mozilla.org/show_bug.cgi?id=1276240

**Comment by facebook-github-bot-6** [comment]
@jimfb updated the pull request.

**Pull request merged**